File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/VpsTrafficResponse.js
goog.provide('API.Client.VpsTrafficResponse');

/**
 * VPS Traffic Information
 * @record
 */
API.Client.VpsTrafficResponse = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.name;

/**
 * @type {!string}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.target;

/**
 * @type {!number}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.interval;

/**
 * @type {!API.Client.VpsTrafficHistoryResponse}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.history;

/**
 * @type {!Date}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.last;

/**
 * @type {!Array<!Date>}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.times;

/**
 * @type {!API.Client.VpsTrafficTotalsResposne}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.totals;

/**
 * @type {!API.Client.VpsTrafficUsageResponse}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.usage;

/**
 * @type {!Array<!API.Client.VpsTrafficDataSectionResponse>}
 * @export
 */
API.Client.VpsTrafficResponse.prototype.data;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/VpsTrafficResponse.scala
package io.swagger.server.model

import java.util.Date

/**
 * = VpsTrafficResponse =
 *
 * VPS Traffic Information
 *
 * @param name 
 * @param target 
 * @param interval 
 * @param history 
 * @param last 
 * @param times 
 * @param totals 
 * @param usage 
 * @param data 
 */
case class VpsTrafficResponse (
  name: String,
  target: String,
  interval: Int,
  history: VpsTrafficHistoryResponse,
  last: Date,
  times: List[Date],
  totals: VpsTrafficTotalsResposne,
  usage: VpsTrafficUsageResponse,
  data: List[VpsTrafficDataSectionResponse]
)

